app/renderer/actions/Inspector.js

```diff
@@ -692,7 +721,7 @@ export function keepSessionAlive () {
 
 export function callClientMethod (params) {
   return async (dispatch, getState) => {
-    const {driver, appMode, mjpegScreenshotUrl} = getState().inspector;
+    const {driver, appMode, mjpegScreenshotUrl, isRefreshingSource} = getState().inspector;
```

Comment:
can we change this to `shouldRefreshSource` or `isSourceRefreshOn`? `isRefreshingSource` sounds like a state variable tracking whether the source is in the process of being refreshed, which I don't think is what you intend.

Reply:
Yup, I see what you mean. Updated to `isSourceRefreshOn`